src/services/pd_services.js

```diff
@@ -0,0 +1,29 @@
+import repo from "../core/pd_repo.js";
+
+import dir from "../schemas/pd_schemas.js";
+
+class services {
+  constructor() {
+    this.getData = repo.getData;
+    this.addData = repo.addData;
+    this.updateData = repo.updateData;
+    this.deleteData = repo.deleteData;
+  }
+  get = async () => {
+    return await this.getData(dir);
```

Comment:
use 
repo.getData(dir);
here no need of variables in constructor

Reply:
done.